MatrixSDK.podspec

```diff
@@ -45,7 +45,7 @@ Pod::Spec.new do |s|
       ss.dependency 'OLMKit', '~> 3.2.5'
       ss.dependency 'Realm', '10.27.0'
       ss.dependency 'libbase58', '~> 0.1.4'
-      ss.dependency 'MatrixSDK/CryptoSDK'
+      ss.dependency 'MatrixSDKCrypto', '0.2.0', :configurations => ["DEBUG", "RELEASE"], :inhibit_warnings => true
```

Comment:
I dont know if this is a coocapods bug, but I have to keep the `configuration` section in (even though it includes all configurations) if I also have the `inhibit_warnings` in

Reply:
Odd, but if that's how it has to be its not going to bother anyone :)